app/services/claim_fast_tracking/max_rating_annotator.rb

```diff
@@ -1,11 +1,32 @@
 # frozen_string_literal: true
 
+require 'virtual_regional_office/client'
+
 module ClaimFastTracking
   class MaxRatingAnnotator
     def self.annotate_disabilities(rated_disabilities_response)
-      tinnitus = ClaimFastTracking::DiagnosticCodes::TINNITUS
-      rated_disabilities_response.rated_disabilities.each do |disability|
-        disability.maximum_rating_percentage = 10 if disability.diagnostic_code == tinnitus
+      return unless Flipper.enabled?(:disability_526_maximum_rating_api)
```

Comment:
This ends up as a no-op if the feature flag is disabled. It would be more useful to keep the previous implementation as a fallback if the flag is disabled, so that the flag switches between the two implementations.

Reply:
@nanotone Should we also fall back to the hardcoded value if the api call fails or has a null response or empty ratings array?